core/src/classes/codeConfig.ts

```diff
@@ -361,20 +361,26 @@ export async function getParentIds(
     providedIds.push(`property:${configObject["bootstrappedProperty"].id}`);
   }
 
-  // - query property with mustache dependency
-  if (
-    cleanClass(configObject) === "property" &&
-    configObject["options"] &&
-    configObject["options"]["query"]
-  ) {
-    const mustachePrerequisiteIds =
-      await MustacheUtils.getMustacheVariablesAsPropertyIds(
-        configObject["options"]["query"],
-        otherConfigObjects
-      );
-    prerequisiteIds.push(
-      ...mustachePrerequisiteIds.map((p) => `property:${p}`)
-    );
+  // - property with mustache dependency
+
+  if (configObject["options"]) {
+    for (const [k, v] of Object.entries(configObject["options"])) {
+      if (
+        cleanClass(configObject) === "property" &&
+        typeof v === "string" &&
+        v.includes("{{") &&
```

Comment:
I think this is a good enough check, but we may want to try/catch around `MustacheUtils.getMustacheVariablesAsPropertyIds()` in the future